### PR TITLE
Feature/iac

### PR DESCRIPTION
**Pull request by sejalv**

### CODE REVIEW

File: 06-best-practices/code/infrastructure/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## IaC w/ Terraform
+
+### Project infrastructure modules in GCP:
```
Comment (alexeygrigorev) [inline 06-best-practices/code/infrastructure/README.md:3]: AWS probably, not GCP

Comment (alexeygrigorev) [reply 06-best-practices/code/infrastructure/README.md:3]:
```
06-best-practices/code/infrastructure/README.md
```
You can probably put it one level up? e.g. `06-best-practices/iac`

Comment (sejalv) [reply 06-best-practices/code/infrastructure/README.md]: Yes, will do, once the README is ready for iac.

File: 06-best-practices/code/infrastructure/vars/prod.tfvars [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// Absolute path?
+lambda_function_local_path="../lambda_function.py"
```
Comment (alexeygrigorev) [inline 06-best-practices/code/infrastructure/vars/prod.tfvars:2]: Why do we need it and the path to the dockerfile? Isn't just the registry URI sufficient?

Comment (sejalv) [reply 06-best-practices/code/infrastructure/vars/prod.tfvars:2]: The lambda infra in TF fails if the config is expecting an Image source and URI, but if the URI doesn't exist. So we either bootstrap a dummy image in the ECR repo to link the lambda with, or we use this one and then update the environment vars in the CI/CD pipeline

**Pull request closed without merging**